src/views/ReturnDetails.vue

```diff
@@ -3,14 +3,21 @@
     <ion-header :translucent="true">
       <ion-toolbar>
         <ion-back-button default-href="/" slot="start"></ion-back-button>
-        <ion-title>{{ $t("Return Details") }}</ion-title>
+        <ion-title>{{ current.externalId }}</ion-title>
       </ion-toolbar>
     </ion-header>
 
     <ion-content>
       <main>
         <ion-item lines="none">
-          <h1>{{ $t("Shipment ID") }}: {{ current.shipmentId }}</h1>
+          <ion-list class="ion-text-center">
+            <ion-title>{{ $t("Order: <order name>") }}</ion-title>
+            <ion-label>{{ $t("Customer: <customer name>") }}</ion-label>
+          </ion-list>
+          <ion-item slot="end" lines="none">
+            <ion-badge :color="statusColors[current.statusDesc]" slot="end">{{ current.estimatedArrivalDate ? ($filters.formatDate(current.estimatedArrivalDate)) : current.statusDesc }}</ion-badge>
```

Comment:
I think we can display the estimatedArrivalDate without having any specific badge color, because in the current case for different status the badge will have different colors and showing date in different colors for shipments might not be a good idea. WDYT @dt2patel Sir?

Reply:
I think we can get rid of ETA from the returns list page all together because the user is most likely scanning in the shipment so they don't need to find it by date.

Reply:
Ok Sir